### ISSUE TITLE
"All Tile Colors" missing colors

### ISSUE THREAD

**Issue by b321gonzo** [debkbanerji/lego-art-remix#81]
"All Tile Colors" option is missing colors, unless I'm misunderstanding the option.

The way I discovered this was when selecting "All Tile Colors" my image didn't look quite right so I did add existing set and did The Art Project set and it added 4 more colors. I'm not sure if it's missing any other colors but the 4 colors the Art Project list had that All Tile Colors doesn't are:
- Dark Azure
- Light Aqua
- Bright Light Yellow
- Bright Green

**Comment by b321gonzo** [comment]
Lots of Dots (round) adds Dark Turquoise 
Extra Dots Series 6 adds Yellowish Green

**Comment by debkbanerji** [comment]
Good catch - fixed in 2022.7.16.  The all tile colors option is annoying to maintain, but it's useful enough that it can't really be removed :(

**Comment by b321gonzo** [comment]
The Globe set adds **_green_** 1x1 round tiles and there is a new Dots patch set that adds **_medium lavender_** 1x1 round tiles, though both are expensive to buy since they are only in those sets

**Comment by b321gonzo** [comment]
And the Orchid set adds **_dark green_** 1x1 round tilez
